.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
We should remove the `eslintConfig` property from the package.json and copy some of the overridden rules to this file.

Also the extends can be: `require.resolve('kcd-scripts/eslint')`

Reply:
Ah it was in the package.json. I was wondering where that was coming from.

>Also the extends can be: require.resolve('kcd-scripts/eslint')

:+1: